packages/thumbnail-grid-block/src/components/Thumbnail/RichTextEditors.tsx

```diff
@@ -0,0 +1,66 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { Thumbnail } from '../../types';
+import { RichTextEditor, convertToRteValue } from '@frontify/guideline-blocks-shared';
+import { captionPlugins, titlePlugins } from '../../helper/plugins';
+import { Asset } from '@frontify/app-bridge';
+import { DesignTokens, TextStyles } from '@frontify/fondue';
+import { useMemo } from 'react';
+
+type RichTextEditorsProps = {
+    designTokens: DesignTokens;
+    isEditing: boolean;
+    updateItemWith: (type: keyof Thumbnail, value: string | Asset[], id?: string) => void;
+    id?: string;
+    title?: string;
+    description?: string;
+};
+
+export const RichTextEditors = ({
+    designTokens,
+    isEditing,
+    updateItemWith,
+    id,
+    title,
+    description,
+}: RichTextEditorsProps) => {
+    const memoizedTitle = useMemo(
+        () => (
+            <div className="[&>div>div>*]:!tw-mt-0">
+                <RichTextEditor
+                    id={`${id}-title`}
+                    designTokens={designTokens}
+                    isEditing={isEditing}
+                    onBlur={(value) => updateItemWith('title', value, id)}
+                    value={title ?? convertToRteValue(TextStyles.ELEMENT_HEADING3)}
+                    placeholder="Add a title"
+                    plugins={titlePlugins}
+                    updateValueOnChange
+                />
+            </div>
+        ),
+        [id, title, designTokens, isEditing, updateItemWith]
+    );
+    const memoizedDescription = useMemo(
+        () => (
+            <RichTextEditor
+                id={`${id}-description`}
+                designTokens={designTokens}
+                isEditing={isEditing}
+                value={description ?? convertToRteValue()}
+                plugins={captionPlugins}
+                onBlur={(value) => updateItemWith('description', value, id)}
+                placeholder="Add a description"
+                updateValueOnChange
```

Comment:
Oh, good to know! Actually it is needed atm, as otherwise when changing the RTE of the placeholder and then blur. It will keep the RTE value for the new created item AND the placeholder. I will check in the next UX review cycle if there is a workaround for it (see checklist `CODE CHECKS` [here](https://app.clickup.com/t/8677h51hv)).